src/Simplex/Chat.hs

```diff
@@ -1072,6 +1083,14 @@ processChatCommand' vr = \case
               pure (e_, map aConnId conns)
     CTLocal -> pure $ chatCmdError (Just user) "not supported"
     CTContactRequest -> pure $ chatCmdError (Just user) "not supported"
+  APIMarkConversationDeleted cRef@(ChatRef cType _chatId) deleted -> withUser $ \user -> case cType of
```

Comment:
when deleting conversation, if contact is not kept, UI would call APIDeleteChat instead